client.go

```diff
@@ -99,6 +99,8 @@ type Client interface {
 	LRSRAdd(lr string, ip_prefix string, nexthop string, output_port []string, policy []string, external_ids map[string]string) (*OvnCommand, error)
 	// Delete LRSR with given ip_prefix on given lr
 	LRSRDel(lr string, ip_prefix string) (*OvnCommand, error)
+	// Delete LRSR with given ip_prefix, nexthop and policy on given lr
+	LRSRDelEx(lr string, prefix string, nexthop, policy *string) (*OvnCommand, error)
```

Comment:
Sorry, one more comment here. We should add all parameters including output_port here, as optional parameter, i.e. compare if it is not nil. There can be routes with same prefix, nexthop, policy, but different output_ports. This API should allow user to delete just one of them.

Reply:
Why a need for new API instead of extending the existing LRSRDel? Default can always be delete by prefix if nexthop, policy, etc are not pass.

Reply:
@noah8713 we want to selectively delete route with the same prefix but different nexthop, policy.

Reply:
@noah8713 Basically, to avoid breaking existing client for compatibility. If there is no other user using this API, it is better to just change the existing API. @jxiaobin maybe @noah8713 's suggestion makes sense because we just discussed in this PR regarding fixing the API LRSRAdd, which would have to break the existing API users.